.github/workflows/build_pytorch_wheels.yml

```diff
@@ -21,12 +24,54 @@ permissions:
 
 jobs:
   build_pytorch_wheels:
-    name: ${{ inputs.families }} | Python ${{ inputs.python_version }}
-    runs-on: ubuntu-22.04
+    name: Build PyTorch Wheels | ${{ inputs.family }} | Python ${{ inputs.python_version }}
+    runs-on: ${{ github.repository_owner == 'ROCm' && 'azure-linux-scale-rocm' || 'ubuntu-24.04' }}
     container:
       image: ghcr.io/rocm/therock_build_manylinux_x86_64:main
     env:
       OUTPUT_DIR: /__w/TheRock/TheRock/output
+
     steps:
       - name: Checkout
         uses: actions/checkout@v4
+
+      - name: Configure Git Identity
+        run: |
+          git config --global user.name "no-body"
+          git config --global user.email "nobody@amd.com"
+
+      - name: Checkout PyTorch Source Repos
+        run: |
+          echo "Checking out PyTorch repositories..."
+          ./external-builds/pytorch/pytorch_torch_repo.py checkout
+          ./external-builds/pytorch/pytorch_audio_repo.py checkout
+          ./external-builds/pytorch/pytorch_vision_repo.py checkout
+
+      - name: Build PyTorch Wheels
+        run: |
+          echo "Building PyTorch wheels for ${{ inputs.target }}"
+          ./external-builds/pytorch/build_prod_wheels.py build \
+            --find-links "https://therock-nightly-python.s3.us-east-2.amazonaws.com/${{ inputs.family }}/index.html" \
+            --install-rocm \
+            --clean \
+            --output-dir ${{ env.OUTPUT_DIR }}/packages/dist \
```

Comment:
When passing a directory name like `${{ env.OUTPUT_DIR }}/packages/dist` between job steps, prefer to store it in a variable like `PACKAGE_DIST_DIR`